mocks/mocks.py

```diff
@@ -15,12 +15,39 @@
 
 import ccdproc
 
+import pynbody
+
 import gunagala
 from gunagala.utils import ensure_unit
 
 from mocks.utils import load_yaml_config
 
 
+def read_gadget(data_path):
+    """
+    Reads the raw output of the Genesis simulations
+
+    Parameters
+    ----------
+    data_path : str
+        The path to the data.
+
+    Returns
+    -------
+    particle_pos: pynbody.array.SimArray
+        The positions of the simulation particles (only stars).
```

Comment:
Are you positive these will *always* be stars? Just leave it out to be safe.

Reply:
No, I think it might be changed in the future when we know more about `Pynbody`. So I change the docstring and mention this point.